rpc/tests/utils/client.rs

```diff
@@ -0,0 +1,72 @@
+use anyhow::Result;
+use celestia_rpc::client::{new_http, new_websocket};
+use celestia_rpc::prelude::*;
+use celestia_types::Blob;
+use jsonrpsee::core::client::ClientT;
+use jsonrpsee::core::Error;
+use jsonrpsee::http_client::HttpClient;
+use jsonrpsee::ws_client::WsClient;
+use once_cell::sync::Lazy;
+use std::env;
+use tokio::sync::Mutex;
+
+static LOCK: Lazy<Mutex<()>> = Lazy::new(|| Mutex::new(()));
```

Comment:
can we have this static inside the blob_submit function? that'd make it impossible to be locked anywhere else

Reply:
Also I know `Lazy` is not stable yet, but we could remove the `once_cell` dependency and do it through stabilized `once_lock`
```rs
{
    static LOCK: OnceLock<Mutex<()>> = OnceLock::new();
    let _guard = LOCK.get_or_init(|| Mutex::new(())).lock().await;
    client.blob_submit(blobs).await
}
```